src/aiowaqi/converters.py

```diff
@@ -0,0 +1,168 @@
+"""Conversion functions for AQI values."""
+
+
+def inv_linear(
+    aqi_high: float,
+    aqi_low: float,
+    conc_high: float,
+    conc_low: float,
+    aqi: float,
+) -> float:
+    """Calculate concentration value using linear interpolation.
+
+    Args:
+        aqi_high: Upper AQI breakpoint
+        aqi_low: Lower AQI breakpoint
+        conc_high: Upper concentration breakpoint
+        conc_low: Lower concentration breakpoint
+        aqi: Input AQI value
+
+    Returns:
+        Calculated concentration value
+
+    """
```

Comment:
So if I understand is that if we have the higher bound of the AQI description we display the highest concentration of that bound and that all the way to the lower part?

Reply:
Yes, we can see this here, for example
![image](https://github.com/user-attachments/assets/0094cd1d-e27a-4c9e-9cc3-a7411a5b2d75)
https://aqicn.org/calculator/#/specie:pm25/concentration:500

Reply:
Is this scale of them linear?

Like, I'd rather use actual formulas than just dividing it into 7 segments (which would give an rough estimation which would be good enough for most, but I am mostly curious if we can go further than that and be accurate)

Reply:
Wow, I just realized that even if they say:
>The above calculator is based on the work from the US EPA Air Now calculator, available at:
[airnow.gov](http://airnow.gov/index.cfm?action=resources.conc_aqi_calc)

On their website, the aqicn scale is not the same than airnow. I will dig deeper into that, my bad. On did all of my tests based on https://www.airnow.gov/aqi/aqi-calculator/ ...

Reply:
Oh no problem. I do have a contact in the air quality business who knows others, so if we have concrete questions on how these things work we can also use that to find out more on this :)